templates/CSI/cephfs/pool.yaml

```diff
@@ -0,0 +1,22 @@
+#################################################################################################################
+# Create a Ceph pool with settings for replication in production environments. A minimum of 3 OSDs on
+# different hosts are required in this example.
+#  kubectl create -f pool.yaml
+#
+#  Origin: https://github.com/rook/rook/blob/master/cluster/examples/kubernetes/ceph/pool.yaml
+#################################################################################################################
+
+apiVersion: ceph.rook.io/v1
+kind: CephBlockPool
```

Comment:
I think this yaml should be under CSI/rbd

Reply:
yep should be in CSI/ folders. thanks

Reply:
Done.